install/ansible/install.sh

```diff
@@ -121,30 +121,36 @@ node_name=$(grep $netmaster $host_inventory | awk '{print $1}' | xargs)
 service_vip=$(ansible $node_name -m setup $ans_opts -i $host_inventory | grep -A 100 ansible_$netmaster_control_if | grep -A 4 ipv4 | grep address | awk -F \" '{print $4}' | xargs)
 
 if [ "$service_vip" == "" ]; then
-	service_vip=$netmaster
+    service_vip=$netmaster
 fi
 
 if [ "$cluster_store" == "" ]; then
-	cluster_store="etcd://localhost:2379"
+    cluster_store="etcd://localhost:2379"
 fi
 
-sed -i.bak "s#.*service_vip.*#\"service_vip\":\"$service_vip\",#g" "$env_file"
-sed -i.bak "s#.*netctl_url.*#\"netctl_url\":\"http://$service_vip:9999\",#g" "$env_file"
-sed -i.bak "s#.*cluster_store.*#\"cluster_store\":\"$cluster_store\",#g" "$env_file"
+sed -i.bak s#'__NETMASTER_IP__'#"$service_vip"#g "$env_file"
+sed -i.bak s#'__CLUSTER_STORE__'#"$cluster_store"#g "$env_file"
+sed -i.bak s#'__DOCKER_RESET_CONTAINER_STATE__'#false#g "$env_file"
+sed -i.bak s#'__DOCKER_RESET_IMAGE_STATE__'#false#g "$env_file"
+sed -i.bak s#'__ETCD_CLEANUP_STATE__'#false#g "$env_file"
+sed -i.bak s#__AUTH_PROXY_LOCAL_INSTALL__#false#g "$env_file"
+sed -i.bak s#__CONTIV_NETWORK_LOCAL_INSTALL__#false#g "$env_file"
```

Comment:
not really a blocker, but you can combine all these into a single sed with -e, the .bak really isn't helping you as it is

Reply:
i used `.bak` was really because it was there ... i agree it is not useful at all.
and multi sed is just to follow the style it was ..